rwgame/DrawUI.cpp

```diff
@@ -101,7 +101,7 @@ void drawPlayerInfo(PlayerController* player, GameWorld* world, GameRenderer* re
 
 	{
 		std::stringstream ss;
-		ss << "@" << std::setw(3) << std::setfill('0')
+		ss << "{" << std::setw(3) << std::setfill('0')
```

Comment:
(See my earlier commits on the commit after this one)

If we introduce a new `GameCharacter` we could have a lookup / enum for common symbols. That would make this code a lot more readable. We'd have to support some kind of `toString()` method OR we simply allow concatenation of `GameCharacter`s (a GameString could simply be a `std::vector<GameCharacter>` which behaves similar to the `std::string` class)

(Note that `GameCharacter` is actually really ambigious as a class name, I don't have any better ideas though)